### PR TITLE
[aws-load-balancer-controller] release controller v2.2.0

### PR DESCRIPTION
**Pull request by kishorj**
### Issue
New chart for controller version v2.0.0
Includes default HA configuration
- replica count 2 
- controller pods priority class system-cluster-critical
- soft pod anti-affinity set so as to not run two controller pods on the same node 

### Checklist
- [x] Added/modified documentation as required (such as the `README.md` for modified charts)
- [x] Incremented the chart `version` in `Chart.yaml` for the modified chart(s)
- [x] Manually tested. Describe what testing was done in the testing section below
- [x] Make sure the title of the PR is a good description that can go into the release notes

### Testing
Updated the IAM policy, updated the CRDs and upgraded the chart. Verified that the controller is able to reconcile service/ingress resources as expected.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: stable/aws-load-balancer-controller/templates/deployment.yaml [hunk 1]
```diff
@@ -135,8 +135,17 @@ spec:
       nodeSelector:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-    {{- with .Values.affinity }}
       affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```
Comment (M00nF1sh) [inline stable/aws-load-balancer-controller/templates/deployment.yaml:140]: i think it's better to be `preferredDuringSchedulingIgnoredDuringExecution`

Comment (kishorj) [reply stable/aws-load-balancer-controller/templates/deployment.yaml:140]: ack

File: stable/aws-load-balancer-controller/templates/webhook.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ webhooks:
     - key: app.kubernetes.io/name
       operator: NotIn
       values:
-      - aws-load-balancer-controller
+      - {{ include "aws-load-balancer-controller.fullname" . }}
```
Comment (M00nF1sh) [inline stable/aws-load-balancer-controller/templates/webhook.yaml:39]: seems we are using `aws-load-balancer-controller.name`

Comment (kishorj) [reply stable/aws-load-balancer-controller/templates/webhook.yaml:39]: thanks. I will update

**Review by M00nF1sh — APPROVED** [review]
/lgtm

**Pull request merged**